lib/pages/settings_dashboard/settings_profile/settings_profile.dart

```diff
@@ -583,6 +603,9 @@ class SettingsProfileController extends State<SettingsProfile>
   void initState() {
     _handleViewState();
     _getCurrentProfile(client);
+    WidgetsBinding.instance.addPostFrameCallback((timeStamp) async {
+      haveMultipleAccountsNotifier.value = await accountsCount > 1;
+    });
```

Comment:
No good for me, UI/UX not smoothly. IMO: You should define state for this. If during get all clients show loading in button


https://github.com/linagora/twake-on-matrix/assets/99852347/f5267e26-15e0-42d9-a63c-798bb3bd7851